### DISCUSSION TITLE
Disable the toggling behaviour when switching to same workspace

### DISCUSSION THREAD

**Discussion by AdityaAparadh** [mylinuxforwork/dotfiles#497] [Q&A] [answered]
When we are on a workspace `x` and try to switch to the same workspace with the keybinding `Super`+`x` , it switches/toggles to the last used workspace. I suppose this is a feature, but this conflicts with my muscle memory, and is especially confusing with animations disabled.

How do I disable this toggling behaviour ? 

I did try to look at `.config/hypr/conf/keybindings/default.conf`, but `bind = $mainMod, x, workspace, x`  seems to be the correct syntax, which is there by default? What other configuration should I change?

**Comment by Veicm** [comment]
I'm not sure if this could be the problem but you have to use the number keys above the keyboard and not the on the number pad even when num lock is active. Maybe this causes the issue and if not could you tell me your OS version.

**Reply by AdityaAparadh** [reply]
Nope, I'm using the number keys above the keyboard.

So that is not supposed to be the default behaviour? Hmm

I'm using Arch Linux with the latest default stable kernel ( `linux` package). I did have a previous Hyprland  config (not ml4w), and when I reverted, it works fine, and has exact same keybindings.
Pretty sure the issue lies somewhere in `.config/hypr`

**Comment by AdityaAparadh** [comment, marked as answer]
Found it, it was just a setting in `.config/hypr/config/layout/default.conf`  😅
There we can turn `binds.workspace_back_and_forth` to `false`
